rust/src/lang/mod.rs

```diff
@@ -22,14 +22,16 @@ pub enum TranslationIds {
     PushNotificationInvitationAcceptedBody,
     UserAlreadyInEmergency,
     UserAlreadyInNormal,
+    UserNotInEmergency,
     NormalModePushNotificationBody,
     EmergencyModePushNotificationBody,
-    PushNotificationActionView
+    PushNotificationActionView,
+    InvalidHistoricalLocationStartTime
 }
 
 
-pub fn get_dictionary(language: String) -> &'static HashMap<TranslationIds, &'static str> {
-    match language.as_str() {
+pub fn get_dictionary(language: &str) -> &'static HashMap<TranslationIds, &'static str> {
```

Comment:
There is a slight difference in definition here between glossary and dictionary. glossary is like a small dictionary for a specific subject so I think that it is more appropriate for this function
```suggestion
pub fn get_glossary(language: &str) -> &'static HashMap<TranslationIds, &'static str> {
```